packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -13,7 +13,8 @@ import { SharedCounter } from "@fluidframework/counter";
 import { type IDirectory, SharedDirectory, SharedMap } from "@fluidframework/map";
 import { SharedMatrix } from "@fluidframework/matrix";
 import { SharedString } from "@fluidframework/sequence";
-import { SharedTree, type ISharedTree, encodeTreeSchema } from "@fluidframework/tree";
+import type { ISharedTree } from "@fluidframework/tree/internal";
+import { SharedTree, encodeTreeSchema } from "@fluidframework/tree/internal";
```

Comment:
Just surprised to see these are all internal now. Are we sure not even ISharedTree should be public?